src/components/DashboardNavigation/DashboardNavigation.js

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import DashboardNavigationStyle from "./DashboardNavigationStyle";
+
+export default function DashboardNavigation() {
+  const isSettingsPage = window.location.pathname.includes("settings");
+  const isUsersPage = window.location.pathname.includes("users");
+  const isTicketsPage = !isSettingsPage && !isUsersPage;
```

Comment:
I think it might be useful to use the React-router's [NavLink component](https://reacttraining.com/react-router/web/api/NavLink) here - it allows you to set custom style/className to link when certain route is matched.

Reply:
done